cli/picker/picker.go

```diff
@@ -0,0 +1,101 @@
+package picker
+
+import (
+	"bytes"
+	"os"
+	"strings"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/arg"
+	"github.com/buildbuddy-io/buildbuddy/cli/bazelisk"
+	"github.com/buildbuddy-io/buildbuddy/cli/log"
+	"github.com/manifoldco/promptui"
+)
+
+func HandlePicker(args []string) []string {
```

Comment:
nit: check whether we're connected to a tty, to handle corner cases of `bb run 2>/dev/null` and `echo foo | bb run`

```suggestion
func HandlePicker(args []string) []string {
	// If not running interactively, we can't show a prompt.
	if !terminal.IsTTY(os.Stdin) || !terminal.IsTTY(os.Stderr) {
		return args
	}

```

Reply:
(err... this should probably be moved lower down, to guard just the code that tries to show a prompt -- since the `build` / `test` commands don't need a prompt

Reply:
Great call, done!